llvm/lib/MCCAS/MCCASObjectV1.cpp

```diff
@@ -3016,43 +3021,41 @@ Error DIEToCASConverter::convertImpl(DWARFDie &DIE, DIEDataWriter &DIEWriter,
       DistinctWriter.writeULEB128(getEndOfDIESiblingsMarker());
       break;
     }
-
-    // FIXME: don't use recursion.
```

Comment:
We should keep this comment, the CAS creation is still recursive. While the performance doesn't quite matter for CAS creation, stack overflow is still a concern

Reply:
Sure that is fair